packages/kafka-iam-auth/src/index.ts

```diff
@@ -54,11 +54,11 @@ const errorEventsListener = (consumer: Consumer): void => {
 
 const kafkaEventsListener = (consumer: Consumer): void => {
   consumer.on(consumer.events.DISCONNECT, () => {
-    logger.info(`Consumer has disconnected.`);
+    logger.debug(`Consumer has disconnected.`);
   });
 
   consumer.on(consumer.events.STOP, (e) => {
-    logger.info(`Consumer has stopped ${JSON.stringify(e)}.`);
+    logger.debug(`Consumer has stopped ${JSON.stringify(e)}.`);
   });
```

Comment:
These listeners have the sole logging purpose, if the log is different from Debug is useless creating them.